images/jenkins/config/template.apko.yaml

```diff
@@ -1,10 +1,5 @@
 contents:
   packages:
-    - glibc-locale-en
-    - bash
-    - coreutils
-    - busybox
-    - tzdata
```

Comment:
I see that these are provided as a runtime dependency over in the main Jenkins package: https://github.com/wolfi-dev/os/blob/7b4049a1440c217c0b0e69cfced2e05deb6181d4/jenkins.yaml#L10-L14, so +1 for pulling these out!